tests/integration/mutate.spec.ts

```diff
@@ -1,4 +1,4 @@
-import * as dgraph from "../../src";
+//import * as dgraph from "../../src";
```

Comment:
If this line isn't needed simply delete it. We don't want code hanging around just in case, it'll stay there in the git version history until needed

Reply:
Yes, this line is not required but I was not sure if I should remove it. 
ts-lint was screaming at this. I'll do it.

Reply:
done